db/schema.rb

```diff
@@ -393,5 +393,6 @@
     t.index ["user_id"], name: "index_web_push_subscriptions_on_user_id"
   end
 
+  add_foreign_key "anonymous_blocks", "users", column: "target_user_id"
```

Comment:
?¿?

Reply:
Rails keeps creating this every time I run migrations. Is it the index on anonymous blocks that creates this foreign key and we just forgot to commit this?

No idea where exactly it'd come from elseway.

Reply:
as long as the tests pass it should be fine ;-) …

Reply:
Anonymous blocking and unblocking also works so it should be okay.